documentation/docs/concepts/decentralized_identifiers/update.mdx

```diff
@@ -4,37 +4,41 @@ sidebar_label: Update
 description: How DID Documents can be manipulated and how updates should be published
 image: /img/Identity_icon.png
 keywords:
-- Documents
-- DID
-- Tangle
-- Update
-- Publish
+  - Documents
+  - DID
+  - Tangle
+  - Update
+  - Publish
 ---
-import account_manipulate_did_rs from '!!raw-loader!../../../../examples_legacy/account/manipulate_did.rs';
-import CodeSnippet from '../../../src/components/CodeSnippetComponent'
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
-import CodeBlock from '@theme/CodeBlock';
 
+import account_manipulate_did_rs from "!!raw-loader!../../../../examples_legacy/account/manipulate_did.rs";
+import update_did_rust from "!!raw-loader!../../../../examples/0_basic/1_update_did.rs";
+import update_did_node from "!!raw-loader!../../../../bindings/wasm/examples/src/ex1_update_did.ts";
+import CodeSnippet from "../../../src/components/CodeSnippetComponent";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import CodeBlock from "@theme/CodeBlock";
 
-DID Documents can be extended by adding [Verification Methods](https://www.w3.org/TR/did-core/#verification-methods), [Services](https://www.w3.org/TR/did-core/#services) and custom properties.
+DID Documents can be extended by adding [verification methods](https://www.w3.org/TR/did-core/#verification-methods), [Services](https://www.w3.org/TR/did-core/#services) and custom properties.
```

Comment:
Link to the sections which then should link out to the specification. Also this article does not cover custom properties other than a single sentence under `Services`, should it have a proper section?

```suggestion
DID Documents can be extended by adding [verification methods](#verification-methods), [services](#services) and custom properties.
```